pages/glossary/decorative.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Decorative
 key: decorative
+unambiguous: false
```

Comment:
This is used in the examples description of 2 rules (which is OK).

This is used in the Expectation of [Image has accessible name](https://act-rules.github.io/rules/23a2a8#expectation), but as part of the expression "marked as decorative" which is unambiguously defined there. So this is OK.

We might want to create a full definition for "marked as decorative", but this would be the topic of another PR.